datahub-web-react/src/app/ingest/source/builder/SelectTemplateStep.tsx

```diff
@@ -104,6 +104,18 @@ export const SelectTemplateStep = ({ state, updateState, goTo, cancel, ingestion
             source.name.toLocaleLowerCase().includes(searchFilter.toLocaleLowerCase()),
     );
 
+    filteredSources.sort((a, b) => {
+        if (a.name === 'custom') {
+            return 1;
+        }
+
+        if (b.name === 'custom') {
+            return -1;
+        }
```

Comment:
this is just to push the "custom" button to the bottom, right?

Reply:
Yes, exactly